### DISCUSSION TITLE
Retrieve open milestones for a repo

### DISCUSSION THREAD

**Discussion by isharailanga** [community/community#24458] [Apps, API and Webhooks]
<p>I want to retrieve all the open milestones for a given repo. What modificatins should I do to the below query.</p>
<pre><code>query{
  organization(login: "wso2") {
    repository(name: "product-apim") {
      name
      url
      milestones{
        edges {
          node {
            id,
            title
          }
        }
      }
    }
  }
}
</code></pre>

**Comment by isharailanga** [comment, marked as answer]
<pre><code>query{
  organization(login: "wso2") {
    repository(name: "product-apim") {
      name
      url
      milestones (first: 100,
      states: [OPEN],
       orderBy: {field: CREATED_AT, direction: DESC},
       ){
        edges {
          node {
            id,
            title
          }
        }
      }
    }
  }
}
</code></pre>
<p>Have to specify first/last value for pagination.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/isharailanga">@isharailanga</a>, Thanks for coming back to share your solution. In the meantime, we’re looking forward to seeing you around!</p>
